File: apollo/src/main/java/io/muun/apollo/domain/errors/InsufficientFundsError.java
package io.muun.apollo.domain.errors;


import io.muun.apollo.external.UserFacingErrorMessages;

public class InsufficientFundsError extends UserFacingError {

    public InsufficientFundsError() {
        super(UserFacingErrorMessages.INSTANCE.insufficientFunds());
    }
}

File: android/apolloui/src/main/java/io/muun/apollo/domain/errors/newop/InsufficientFundsError.kt
package io.muun.apollo.domain.errors.newop


import io.muun.apollo.data.external.UserFacingErrorMessages
import io.muun.apollo.domain.errors.UserFacingError

class InsufficientFundsError : UserFacingError(
    UserFacingErrorMessages.INSTANCE.insufficientFunds()
)
